opentelemetry-tracing/opentelemetry-java-tracing/spring-boot-kotlin/README.md

```diff
@@ -21,17 +21,16 @@ $ ./start.sh
 Here are some environment variables you might need to set to configure the Java
 agent:
 ```
-$ # Exporter configuration - default is Zipkin exporter with endpoint URL http://localhost:9080/v1/trace"
-$ # Eg. set OTLP exporter - compatible with SignalFX SmartAgent
-$ export OTEL_EXPORTER=otlp
+$ # Exporter configuration - default is the Jaeger Thrift exporter with endpoint URL "http://localhost:9080/v1/trace"
+$ # Eg. to set up the application to use the OTLP exporter - compatible with SignalFX SmartAgent
+$ export OTEL_TRACES_EXPORTER=otlp
 
 $ # Exporter endpoint URL - each exporter has own, specific property name and the default value  
-$ # Eg. set OTLP exporter URL - default is localhost:55680
-$ export OTEL_OTLP_ENDPOINT=localhost:9411
+$ # Eg. to set up the OTLP exporter URL - default is http://localhost:4317
+$ export OTEL_OTLP_ENDPOINT=http://localhost:12345
```

Comment:
I bet the goal here is to show how a different port can be specified, but I would keep the default 4317 in this example `export` statement as well, because it will get copy/pasted.

Reply:
I thought the point here was to show how to change the default, though?